### DISCUSSION TITLE
Export as *.STL

### DISCUSSION THREAD

**Discussion by wells5150** [sn4k3/UVtools#777] [Ideas]
It would streamline processes to export as an *.STL file. My printer doesn't recognize files from UV tools. *.CTB files when opened in chitubox render errors for me.

**Comment by sn4k3** [comment]
UVtools can export to stl: Layer actions -> Export to mesh.
However that is for recreation only. Should not be used anywhere to slice or feed to printer as original model is lost and generate a ton of triangles.

What is your printer?

**Reply by wells5150** [reply]
I shall try again. I use Phrozen Sonic Mighty 8K.

**Comment by Hazoooka** [comment]
Do you simply "save" the CTB or use the "convert to" function?

**Comment by wells5150** [comment]
convert
